app/src/main/java/org/mozilla/fenix/Experiments.kt

```diff
@@ -4,27 +4,8 @@
 
 package org.mozilla.fenix
 
-import android.content.Context
-import mozilla.components.service.fretboard.ExperimentDescriptor
-
-const val EXPERIMENTS_JSON_FILENAME = "experiments.json"
-const val EXPERIMENTS_BASE_URL = "https://firefox.settings.services.mozilla.com/v1"
-const val EXPERIMENTS_BUCKET_NAME = "main"
-// collection name below, see https://bugzilla.mozilla.org/show_bug.cgi?id=1523395 for ownership details
-const val EXPERIMENTS_COLLECTION_NAME = "fenix-experiments"
-
 object Experiments {
-    val AATestDescriptor = ExperimentDescriptor("AAtest")
-    // application services flag to disable the Firefox Sync syncManager
-    val asFeatureSyncDisabled = ExperimentDescriptor("asFeatureSyncDisabled")
-    // application services flag to disable Firefox Accounts pairing button.
-    val asFeatureFxAPairingDisabled = ExperimentDescriptor("asFeatureFxAPairingDisabled")
-    // application services flag to disable Firefox Accounts WebChannel integration.
-    val asFeatureWebChannelsDisabled = ExperimentDescriptor("asFeatureWebChannelsDisabled")
+    const val asFeatureWebChannelsDisabled = "asFeatureWebChannelsDisabled"
+    const val asFeatureSyncDisabled = "asFeatureSyncDisabled"
+    const val asFeatureFxAPairingDisabled = "asFeatureFxAPairingDisabled"
```

Comment:
It's worth noting that @colintheshots reached out to the devs who created these flags to let them know they won't be remotely toggleable after this change.